tools/runners/run-mongo.js

```diff
@@ -213,8 +213,7 @@ if (process.platform === 'win32') {
             var foundPort = parseInt(m[2], 10);
             var foundPath = m[3];
 
-            if ( (! port || port === foundPort) &&
-                 (! dbDir || dbDir === foundPath)) {
+            if (! dbDir || dbDir === foundPath) {
```

Comment:
Without this change, the only `mongod` processes we'll find are those using the same port we're trying to use, which is not the case if we've changed the `--port`.

Since we're still checking the `--dbPath`, this search won't return any `mongod` processes that don't belong to this app.